backend/dissemination/search.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import Q
+
+from dissemination.models import General
+
+
+def search_general(
+    names=None,
+    uei_or_eins=None,
+    start_date=None,
+    end_date=None,
+    cog_or_oversight=None,
+    agency_name=None,
+    audit_years=None,
+):
+    query = Q(is_public=True)
```

Comment:
For the future, I think I'd like us to... subclass, decorate, or otherwise find a way to make sure that we have a `PublicQuery` and a... `ProtectedQuery` in the code. The `PublicQuery` always appends `is_public=True`. 

Or, something.

Just looking at this, and thinking that someone could easily/accidentally assign to `query` somewhere in the list below, and we would lose our `is_public` from the query that is being built.

Putting this at the end, right before the query is composed, might be safer for the moment. Going forward, having a way to clearly signal that it is a "public-only query," and enforce it by construction in code, would be good.

I'm going to start an "increment v2" ticket to track things like this.